.github/workflows/ci.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - name: Build
         run: npm run build
         env:
-          NODE_OPTIONS: --max-old-space-size=6144
+          NODE_OPTIONS: --max-old-space-size=8192
```

Comment:
What's the max for our runner? Might as well just jump to something near that?

Reply:
Good idea, it [looks like](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) the runners have 16GB RAM. I'll bump this up to 12288.